com.unity.netcode.gameobjects/CHANGELOG.md

```diff
@@ -32,6 +32,7 @@ Additional documentation and release notes are available at [Multiplayer Documen
 - Fixed issue where NetworkManager would continue starting even if the NetworkTransport selected failed. (#1780)
 - Fixed issue when spawning new player if an already existing player exists it does not remove IsPlayer from the previous player (#1779)
 - Fixed lack of notification that NetworkManager and NetworkObject cannot be added to the same GameObject with in-editor notifications (#1777)
+- `UnityTransport` send queues are now flushed when calling `DisconnectLocalClient` or `DisconnectRemoteClient`. (#1847)
```

Comment:
Most likely these are both nit-picks:
- Could you change how the fix is worded sort of like: "- Fixed issue where send queues were not being flushed when calling `DisconnectLocalClient` or `DisconnectRemoteClient`. (#1847)
- Could you move it to the top in order to preserver a descending order?  (my OCD kicking in)